homeassistant/components/modbus/modbus.py

```diff
@@ -234,6 +235,18 @@ async def async_stop_hub(service: ServiceCall) -> None:
 
     async def async_restart_hub(service: ServiceCall) -> None:
         """Restart Modbus hub."""
+        async_create_issue(
+            hass,
+            DOMAIN,
+            "deprecated_restart",
+            breaks_in_ha_version="2024.11.0",
+            is_fixable=False,
+            severity=IssueSeverity.WARNING,
+            translation_key="deprecated_restart",
+        )
+        _LOGGER.warning(
+            "`modbus.restart`: is deprecated and will be removed in version 2024.11"
```

Comment:
I'd remove the colon in the message.

Reply:
https://github.com/home-assistant/core/pull/116399